tests/phpunit/integration/Core/Authentication/AuthenticationTest.php

```diff
@@ -888,6 +888,25 @@ function ( $preempt, $args, $url ) use ( $google_proxy ) {
 		$this->assertFalse( apply_filters( 'googlesitekit_is_feature_enabled', false, 'test.featureTwo' ) );
 	}
 
+	public function test_invalid_nonce_error() {
+		try {
+			Authentication::invalid_nonce_error( 'log-out' );
+		} catch ( WPDieException $exception ) {
+			$this->assertStringStartsWith( 'You are attempting to log out of Test Blog', $exception->getMessage() );
+			return;
+		}
+		try {
+			Authentication::invalid_nonce_error( 'googlesitekit_proxy_foo_action' );
+		} catch ( WPDieException $exception ) {
+			$this->assertEquals( 'The link you followed has expired.</p><p><a href="http://example.org/wp-admin/admin.php?page=googlesitekit-splash">Please try again.</a>', $exception->getMessage() );
+			return;
+		}
```

Comment:
Let's split it into two tests.

Reply:
https://github.com/google/site-kit-wp/pull/3976/commits/8fe8aba732485d21e1219f969c254c81d018d0da